src/doc/rustdoc/src/how-to-read-rustdoc.md

```diff
@@ -0,0 +1,101 @@
+which shows contextual information about the current entry.
+The rest of the is taken up by the search interface at the top
+and the documentation for the current item below that.
+
+## The Item Documentation
+
+The majority of the screen is taken up with the main event:
+the documentation text for the item currently being viewed.
+The main title is the type and name of the item,
+such as "Struct `std::time::Duration`",
+along with a button to collapse or expand the top-level documentation for that
+item (`[+]` or `[-]`),
+a link to the source code (`[src]`),
+if [configured](the-doc-attribute.html#html_no_source),
+and the version in which the item became stable,
+if it's a stable item in the standard library.
+
+Below this is the main documentation for the item,
+including a definition or function signature if appropriate,
+followed by a list of fields or variants for structs, unions, and enums.
+Finally, the page lists associated functions and trait implementations,
+including automatic and blanket implementations that `rustdoc` knows about.
+
+### Navigation
+
+Subheadings, variants, fields, and many other things in this documentation
+are anchors and can be clicked on and deep-linked to,
+which is a great way to communicate exactly what you're talking about.
+The typograpical character "§" appears next to lines with anchors on them
+when hovered or given keyboard focus.
+
+## The Navigation Bar
+
+For example, when looking at documentation for the crate root,
+it shows all the crates documented in the documentation bundle,
+and quick links to the modules, structs, traits, functions, and macros available
+from the current crate.
+At the top, it displays a [configurable logo](the-doc-attribute.html#html_logo_url)
+alongside the current crate's name and version,
+or the current item whose documentation is being displayed.
+
+## The Theme Picker and Search Interface
+
+When viewing `rustdoc`'s output in a browser with JavaScript enabled,
+a dynamic interface appears at the top of the page.
+To the left is the theme picker, denoted with a paint-brush icon,
+and the search interface and options appear to the right of that.
+
+### The Theme Picker
+
+Clicking on the theme picker provides a list of themes -
+by default `ayu`, `light`, and `dark` - 
+which are available for viewing.
+
+### The Search Interface
+
+Typing in the search bar instantly searches the available documentation for
+the string entered, with a fuzzy matching algorithm that is tolerant of omitted
+letters but not extra ones.
+
+By default, the search results give are "In Names", meaning that the fuzzy
+match is made against the names of items.
+Matching names are shown on the left, and the first few words of their
+descriptions are given on the right.
+By clicking an item, you will navigate to its particular documentation.
+
+There are two other modes, shown as tabs in the search results pane.
+"In Parameters" searches for the string in the types of parameters to functions,
```

Comment:
It doesn't search when clicking on another tab, it's simply how they're classified.

Reply:
Ah, important distinction to make. Thanks!